cassandra-diagnostics-reporter-riemann/src/main/java/io/smartcat/cassandra/diagnostics/reporter/RiemannReporter.java

```diff
@@ -56,12 +56,12 @@ public void report(Query queryReport) {
             return;
         }
 
-        logger.debug("Sending Query: execTime={}", queryReport.executionTimeInMilliseconds);
+        logger.debug("Sending Query: execTime={}", queryReport.getExecutionTimeInMilliseconds());
         try {
```

Comment:
Here is good example, to me it reads much better `queryReport.executionTimeInMilliseconds()`